hive/src/main/java/io/confluent/connect/storage/hive/HiveUtil.java

```diff
@@ -28,18 +28,25 @@
  */
 public abstract class HiveUtil {
 
-  protected String url;
+  protected final String url;
   protected final HiveMetaStore hiveMetaStore;
   protected final String delim;
 
   public HiveUtil(AbstractConfig connectorConfig, HiveMetaStore hiveMetaStore) {
-    this.url = connectorConfig.getString(StorageCommonConfig.STORE_URL_CONFIG);
+    this(connectorConfig,
+            hiveMetaStore,
+            connectorConfig.getString(StorageCommonConfig.STORE_URL_CONFIG));
```

Comment:
alignment is a bit off. In connectors, tab space is equal to 2 spaces (as opposed to AK checkstyle). 
```suggestion
    this(
        connectorConfig,
        hiveMetaStore,
        connectorConfig.getString(StorageCommonConfig.STORE_URL_CONFIG)
    );
```
(did the editing on GH to give an example. Please make sure it's correct)